charts/juicefs/values.yaml

```diff
@@ -1,5 +1,6 @@
 # This should not be modified in the usual case.
 fullnameOverride: juicefs
+isCe: false
```

Comment:
I suggest using meaningful name, because I'm worried about forgetting the meaning of the name in future.